bindings/qt/sdk.pri

```diff
@@ -497,7 +500,16 @@ else {
             LIBS += $$MEGASDK_BASE_PATH/bindings/qt/3rdparty/libs/libfreeimage.a
         }
         else {
-            LIBS += -lfreeimage
+            LIBS += -lfreeimage$$DEBUG_SUFFIX
+        }
+
+        vcpkg {
+            LIBS += -ljpeg$$DEBUG_SUFFIX -ltiff$$DEBUG_SUFFIX -llibpng16$$DEBUG_SUFFIX \
+            -lIlmImf-2_3$$UNDERSCORE_DEBUG_SUFFIX -lIex-2_3$$UNDERSCORE_DEBUG_SUFFIX -lIlmThread-2_3$$UNDERSCORE_DEBUG_SUFFIX \
+            -lIexMath-2_3$$UNDERSCORE_DEBUG_SUFFIX -lIlmImfUtil-2_3$$UNDERSCORE_DEBUG_SUFFIX -lImath-2_3$$UNDERSCORE_DEBUG_SUFFIX \
+            -llibwebpmux$$DEBUG_SUFFIX -lwebpdecoder$$DEBUG_SUFFIX -lwebpdemux$$DEBUG_SUFFIX -lwebp$$DEBUG_SUFFIX \
+            -ljpegxr$$DEBUG_SUFFIX -ljxrglue$$DEBUG_SUFFIX -lHalf-2_3$$UNDERSCORE_DEBUG_SUFFIX \
```

Comment:
I see freeimage can be used under LGPL or FIPL,  I guess we are using FIPL?  Or, are we treating Freeimage as a single DLL or multiple?    Also will we specify the licence somewhere in MEGAcmd ?   thanks

Reply:
Here's the license note (FIPL):
https://github.com/meganz/sdk/blob/develop/CREDITS.md#freeimage
MEGAcmd would link it as a single static .lib